packages/cadl-ranch-specs/http/dictionary/mockapi.ts

```diff
@@ -14,18 +14,17 @@ interface MockApiGetPut {
  */
 function createModelMockApis(route: string, value: any): MockApiGetPut {
   const url = `/dictionary/${route}`;
-  const body = { property: value };
   return {
     get: mockapi.get(url, (req) => {
       return {
         status: 200,
-        body: json(body),
+        body: json(value),
       };
     }),
     put: mockapi.put(url, (req) => {
-      req.expect.bodyEquals(body);
+      req.expect.bodyEquals(value);
       return {
-        status: 204,
+        status: 200,
```

Comment:
Should we change the spec to return `void ` instead of `OkResponse` would be more consistent with the others

Reply:
changed to use `void` and resumed to `status: 204`